### ISSUE TITLE
Expost  packageName && serviceName

### ISSUE THREAD

**Issue by delaneyj** [hopin-team/twirp-ts#1]
in the `createContext` that gets generated there is 

```ts
packageName: "foo.bar.baz",
serviceName: "Boo",
```

could you please expose on the service ala
```ts
const service = createHaberdasherServer(new Habadasher()); 
const {packageName, serviceName} = service
```

I need this to do dynamic routing for within a server.  (hosting some static files and possible a UI related to generated swagger for the twirp routes.

**Comment by fenos** [comment]
Hi @delaneyj,

Yes! That seems super useful I will expose those 2 properties!

**Comment by delaneyj** [comment]
Awesome, and if you make them `public readonly ` then you'll be able to use throughout the generated functions which mean less magic values!

**Comment by fenos** [comment]
this is now fixed by #4 you can access the 2 properties as suggested above
